cmd/bootstrap/cmd/finalize.go

```diff
@@ -261,7 +265,11 @@ func assembleInternalNodes() []model.NodeInfo {
 
 		// validate every single internal node
 		nodeID := validateNodeID(internal.NodeID)
-		stake := validateStake(stakes[internal.Address])
+		stake, valid := validateStake(stakes[internal.Address])
+		if !valid {
+			log.Error().Msgf("stakes: %v", stakes)
+			log.Fatal().Msgf("internal node %v has no stake. Did you forget to update the node address?", internal.Address)
```

Comment:
should we print out all the node info and not just the address here? i.e. print out all of  internal` instead of just `internal.address`